src/Fcf/Utils.hs

```diff
@@ -16,8 +16,13 @@ module Fcf.Utils
   , TyEq
   , Stuck
   , IsBool(_If)
+  , Case
+  , Match
+  , type (-->)
+  , Any
+  , Else
 
-    -- | From "Data.Type.Bool".
+    -- * From "Data.Type.Bool"
```

Comment:
Changed to title to avoid confusion - previous comment looked like line in `Else` doc separated by a newline.